packages/block-editor/src/components/block-list/block.js

```diff
@@ -284,16 +284,14 @@ function BlockListBlock( {
 			isFirstMultiSelected
 		);
 
-	// Insertion point can only be made visible if the block is at the
-	// the extent of a multi-selection, or not in a multi-selection.
-	const shouldShowInsertionPoint = ! isMultiSelecting && (
-		( isPartOfMultiSelection && isFirstMultiSelected ) ||
-		! isPartOfMultiSelection
-	);
-
-	const shouldRenderDropzone = shouldShowInsertionPoint;
 	const isDragging = isDraggingBlocks && ( isSelected || isPartOfMultiSelection );
 
+	const { position } = useBlockDropZone( {
+		element: wrapper,
+		clientId,
+		rootClientId,
+	} );
```

Comment:
It seems like the dropzone is always active now while previously it was only rendered in some circumstances?

Reply:
Perviously it was ignored for multi selection because block controls would interfere with the selection. Since the controls are now gone from the block DOM, this exception is no longer necessary.